docs/index.rst

```diff
@@ -20,6 +20,8 @@ new transient reported to the IAU in real-time after the transient is
 announced. This information is not provided by any existing transient
 broker service.
 
+Blast is currently developed by the <a href="https://transients.ucsc.edu/">UCSC Transients Team</a>.
+
```

Comment:
Nice work, very close. For html links in the documentation you need:

```
`UCSC Transients Team <https://transients.ucsc.edu/>`_
```

not the html tag! Can you please swap them out :-)

Reply:
Perfect :)